src/components/select/multi-select/components.test-pw.tsx

```diff
@@ -365,6 +424,29 @@ export const MultiSelectCustomColorComponent = (
   );
 };
 
+export const MultiSelectListActionEventComponent = (
+  props: Partial<MultiSelectProps>
```

Comment:
suggestion (non-blocking): you could make `onListAction` `Required` here so it's flagged if someone ever uses it and doesn't pass the callback etc